tutorial/10-spatial-solver.ipynb

```diff
@@ -294,13 +297,13 @@
      "language": "python",
      "metadata": {},
      "outputs": [],
-     "prompt_number": 16
+     "prompt_number": 17
     },
     {
      "cell_type": "markdown",
      "metadata": {},
      "source": [
-      "'x_boundry_conditions', 'y_boundry_conditions', and 'z_boundry_conditions' are the boundry conditions for each face of the cubic mesh.  The entries are as following (x is the array set to the key 'x_boundry_conditions', y to 'y_boundry_conditions' and z to 'z_boundry_conditions'.\n",
+      "'x_boundry_conditions', 'y_boundry_conditions', and 'z_boundry_conditions' are the boundry conditions for each face of the cubic mesh.  The entries are as follows: x is the array set to the key 'x_boundry_conditions', y to 'y_boundry_conditions' and z to 'z_boundry_conditions'.\n",
```

Comment:
On line 309-310 

       "    y[1] = yebc\n",
       "    y[0] = zsbc\n",
       "    y[1] = zebc\n",

Should be 

       "    y[1] = yebc\n",
       "    z[0] = zsbc\n",
       "    z[1] = zebc\n",

Right?